FirebaseRemoteConfig/Sources/RCNDevice.m

```diff
@@ -26,12 +26,17 @@
 #define STR_EXPAND(x) #x
 
 static NSString *const RCNDeviceContextKeyVersion = @"app_version";
+static NSString *const RCNDeviceContextKeyBuild = @"app_build";
 static NSString *const RCNDeviceContextKeyOSVersion = @"os_version";
 static NSString *const RCNDeviceContextKeyDeviceLocale = @"device_locale";
 static NSString *const RCNDeviceContextKeyLocaleLanguage = @"locale_language";
 static NSString *const RCNDeviceContextKeyGMPProjectIdentifier = @"GMP_project_Identifier";
 
 NSString *FIRRemoteConfigAppVersion() {
+  return [[NSBundle mainBundle] infoDictionary][@"CFBundleShortVersionString"];
```

Comment:
Isn't this a breaking change to change the implementation of `FIRRemoteConfigAppVersion`? Is this a copy/paste error?

Reply:
The backend is modified to accept this.

Reply:
Understood, but if someone does an update and their Info.plist doesn't map BundleVersion to ShortVersion the same way the backend does, won't there be issues?

Reply:
The backend uses the CFBundleVersion today for targeting. Developers have requested using the shortversion for targeting. To support existing the existing SDK out there, we will continue to honor the CFBundleVersion. If the build_version is present, it will be used as the CFBundleVersion. (There will also be a console change to support this)